setup.py

```diff
@@ -5,12 +5,17 @@
 from setuptools import setup, find_packages
 import sys
 
-# Only 2.7 is supported.
-if not ((2, 7, 0) <= sys.version_info[:3] < (2, 8)):
-    sys.stderr.write('gmusicapi does not officially support this Python version.\n')
-    # try to continue anyway
-
 dynamic_requires = []
+# Python 2.7 is supported. Python 3 support is experimental
+if sys.version_info[0] > 2:
+    sys.stderr.write("gmusicapi Python 3 support is experimental.\n")
```

Comment:
Ah, yeah, I like that, and it's backwards compatible with what's done here.

How about `warnings.warn("gmusicapi ...", RuntimeWarning)`?